.github/workflows/lint.yml

```diff
@@ -42,15 +42,19 @@ jobs:
           export DEBIAN_FRONTEND=noninteractive
           apt update && apt install ca-certificates openssl -yq
       - uses: actions/checkout@v2
+      - name: Setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.4
       - name: run golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
-          # Caching doesn't work on GH Enterprise w/ self-hosted runners
+          version: v1.43.0
+          skip-go-installation: true
           skip-pkg-cache: true
           skip-build-cache: true
```

Comment:
@andy-paine suggested to disable caching. Can be also done in a separate PR.

Reply:
@johha  enable or disable ? currently it is disabled. Just the Comment for GHE is removed :) .